### DISCUSSION TITLE
Do these variables replace,  system_prompt ,query_wrapper_prompt

### DISCUSSION THREAD

**Discussion by seoeaa** [safevideo/autollm#115] [Q&A] [answered]
Do these variables replace,  system_prompt ,query_wrapper_prompt  instructions in file  templates.py

   system_prompt='....................',
   query_wrapper_prompt='.....................',

query_engine = AutoQueryEngine.from_parameters(
    documents=documents,
    #system_prompt='....................',
    #query_wrapper_prompt='.....................',
    enable_cost_calculator=True,
    llm_params=llm_params,
    service_context_params=service_context_params,
    vector_store_params={
        "vector_store_type": "LanceDBVectorStore",
        "uri": "/tmp/stroyka",  #stroyka ,
        "table_name": "lancedb",
        "nprobs": 20
    },
    query_engine_params={"similarity_top_k": 10},
)

**Comment by talhasarit** [comment, marked as answer]
Hi @seoeaa,

Yes, the `system_prompt` and `query_wrapper_prompt` are used if you set with your desired prompt text in the `AutoQueryEngine` setup. If you prefer to don't set them, just remove and our function will use the defaults in our `templates` module :)
